pyproject.toml

```diff
@@ -3,7 +3,7 @@ name = "python-jsx"
 authors = [{ name = "Tomas Roun", email = "tomas.roun8@gmail.com" }]
 description = "JSX transpiler for Python"
 readme = "README.md"
-requires-python = ">=3.12"
+requires-python = ">=3.10,<3.14"
```

Comment:
If we're gonna add support for more Python versions, I think we should also use them in our CI tests. Currently only 3.12 is tested.

Reply:
That makes total sense. I added the other versions to the CI.